tools/builder/react.js

```diff
@@ -184,9 +182,13 @@ function buildStatic( done ) {
 		};
 
 		try {
-			path = __dirname + '/../../_inc/build/static.js';
+			// normalize path
+			const path = require.resolve( __dirname + '/../../_inc/build/static.js' );
```

Comment:
This is the important change: using `require.resolve()` to normalize the path, so that we delete the correct entry from `require.cache` below.